src/push_gitlab_build_status_to_github_website.sh

```diff
@@ -47,28 +51,28 @@ get_and_export_build_status_to_github_build_status_website_repo() {
 	pipelines=$(curl --header "PRIVATE-TOKEN: $personal_access_token" "http://127.0.0.1/api/v4/projects/$gitlab_username%2F$repository_name/pipelines")
 	
 	# get build status from pipelines
-	job=$(echo $pipelines | jq -r 'map(select(.sha == "'"$branch_commit"'"))')
-	status=$(echo "$(echo $job | jq ".[].status")" | tr -d '"')
+	job=$(echo "$pipelines" | jq -r 'map(select(.sha == "'"$branch_commit"'"))')
+	status=$(cmd "$(echo "$job" | jq ".[].status")" | tr -d '"')
```

Comment:
Could you explain what the difference is between using `echo` and `cmd`?  I'm simply curious :)

Reply:
Previously, it was supplying an `echo` to an `echo` where the second `echo` was a normal *command* so it's better to just use `cmd`.

Reply:
Thank you for your explanation!